CHANGES.md

```diff
@@ -10,6 +10,7 @@
 - Billboards using `imageSubRegion` now render as expected. [#12585](https://github.com/CesiumGS/cesium/issues/12585)
 - Improved scaling of SVGs in billboards [#13020](https://github.com/CesiumGS/cesium/issues/13020)
 - Fixed unexpected outline artifacts around billboards [#13038](https://github.com/CesiumGS/cesium/issues/13038)
+- Fixed image alignment in large billboard collections [#13042](https://github.com/CesiumGS/cesium/issues/13042)
```

Comment:
I think the link is wrong.
Normally we link to the issue we're fixing. In this case, since it's only a partial fix, maybe a link to the PR does make sense?

Reply:
GitHub cleverly resolves any `/issues/<number>/` and `/pull/<number>` to the respective `/pull/` or `/issues/` version, just based on the number. (I.e. this link, even though it (literally) has `/issues/`, is redirected to **this** PR).